### PR TITLE
[ 🎨 ] Redesign and Views

### PR DESCRIPTION
**Pull request by KitaecaCapalot**
Views now semi-functional, redesign almost done with the exception of the new logic buttons. Looking forward for the new logic commits. 

~ "Plus Ultra!" - All Might

### CODE REVIEW

File: BudgetManagementApp/BudgetManagment.Data/Migrations/20240606052519_Initial.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public partial class Initial : Migration
         protected override void Up(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.CreateTable(
-                name: "BudgetCategories",
```
Comment (ectky) [inline BudgetManagementApp/BudgetManagment.Data/Migrations/20240606052519_Initial.cs]: ?

File: BudgetManagementApp/BudgetManagementApp/appsettings.json [hunk 2]
```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=localhost\\SQLEXPRESS;Initial Catalog=BudgetManagementMVC;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=True;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=DESKTOP-074U5IO\\KITAECA;Initial Catalog=BudgetManagementMVC;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=True;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```
Comment (ectky) [inline BudgetManagementApp/BudgetManagementApp/appsettings.json]: .

**Review by ectky — CHANGES_REQUESTED** [review]

File: BudgetManagementApp/BudgetManagementApp/Controllers/RoleController.cs [hunk 3]
```diff
@@ -5,14 +5,17 @@
...
 using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace BudgetManagementApp.Controllers
 {
     [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme, Roles = "Admin, Employee, User")]
     public class RoleController : BaseCrudController<RoleDto, IRoleRepository, IRoleService, RoleEditVM, RoleDetailsVM>
     {
-        public RoleController(IRoleService service, IMapper mapper) : base(service, mapper)
+        public RoleController(IRoleService service, IMapper mapper, IBudgetService budgetService, IRoleService roleService) : base(service, mapper)
```
Comment (ectky) [inline BudgetManagementApp/BudgetManagementApp/Controllers/RoleController.cs]: тези депендънсита защо?

**Review by ectky — CHANGES_REQUESTED** [review]

File: BudgetManagementApp/BudgetManagementApp/Controllers/RoleController.cs (hunk 3 above)
Comment (KitaecaCapalot) [reply BudgetManagementApp/BudgetManagementApp/Controllers/RoleController.cs]: zabrail sum gi 💀

**Review by ectky — APPROVED** [review]

**Pull request merged**